AutonomousDemo/Velocity.cs

```diff
@@ -9,95 +9,50 @@ namespace AutonomousDemo
     public class Velocity
     {
         public double Speed { get; set; } = 0;
-        public Direction Direction { get; set; } //radians
+        public double Direction { get; set; }  //radians
 
-        //public Velocity(double metersTraveled, double time, Direction direction)
-        //{   
-        //    this.Speed =  metersTraveled / time;
-        //    this.Direction = direction;
-        //}
 
-        public Velocity(double speed, Direction direction)
+        public Velocity(double speed, double direction = 1.5708)
```

Comment:
1. Why is your default direction pi/2?
2. If it is going to be that, I would recommend putting "Math.PI / 2.0" instead if you can (because not everybody is going to recognize that as pi/2).